core/store/migrations/migration1604437959/migrate.go

```diff
@@ -3,19 +3,19 @@ package migration1604437959
 import "github.com/jinzhu/gorm"
 
 const up = `
-	CREATE TABLE "pipeline_spec_errors" (
+	CREATE TABLE "job_spec_errors_v2" (
```

Comment:
I think this is allowed since its within the same release? I'm assuming only like if this change straddled a release would it cause problems (and we'd need to do a new migration instead)

Reply:
Personally I tend to agree with you.

Sometimes gives ops headaches but imo that should be expected with develop branch anyway.